packages/orchestration/src/exos/chain-hub.js

```diff
@@ -425,6 +437,20 @@ export const makeChainHub = (zone, agoricNames, vowTools) => {
       }
       return undefined;
     },
+    /**
+     * @param {string} address bech32 address
+     * @returns {CosmosChainInfo}
+     */
+    getChainInfoFromAddress(address) {
```

Comment:
Did you consider `getChainInfoFromPrefix(getBech32Prefix(address))` instead or in addition?

Reply:
I didn't, no.

> in addition

I think we should only have one function for this to reduce info overload.

> instead

I still feel `getChainInfoFromAddress` is the sensible abstraction, but happy to refactor to `getChainInfoFromPrefix` if you and other reviewers are in agreement that its preferable.

Reply:
This starts to look like a product question. `getChainInfoFromAddress` is what #10349 calls for, and I guess it got an OK from product.

I don't feel strongly about it. I guess `getChainInfoFromAddress` is fine by me.